widgets/htdocs/components/95_PDB.js

```diff
@@ -41,10 +41,14 @@ Ensembl.Panel.PDB = Ensembl.Panel.Content.extend({
 
     // Setup external links   
     this.protein_sources = { 
-                             'Pfam'   : 'http://pfam.xfam.org/family/',
-                             'PRINTS' : 'https://www.ebi.ac.uk/interpro/signature/',
-                             'Gene3D' : 'http://gene3d.biochem.ucl.ac.uk/Gene3D/search?mode=protein&sterm='
+                             'Pfam'    : 'http://pfam.xfam.org/family/',
+                             'PRINTS'  : 'https://www.ebi.ac.uk/interpro/signature/',
+                             'Gene3D'  : 'http://gene3d.biochem.ucl.ac.uk/Gene3D/search?mode=protein&sterm=',
+                             'PANTHER' : 'http://www.pantherdb.org/panther/family.do?clsAccession=',
+                             'Smart'   : 'http://smart.embl-heidelberg.de/smart/do_annotation.pl?DOMAIN='
                            };
+    this.sorted_protein_source_names = Object.keys(this.protein_sources).sort();
```

Comment:
You can remove this line by just having $.each(Object.keys(this.protein_sources).sort(),....) if the variable sorted_protein_source_names isn't used again. Just one less long variables :)

Reply:
Thank you for the feedback!
I removed the variable